spec/p2p/v0.34/transport.md

```diff
@@ -0,0 +1,223 @@
+# Transport
+
+The transport establishes secure and authenticated connections with peers.
+
+The transport [`Dial`](#dial)s peer addresses to establish outbound connections,
```

Comment:
I would often find it useful to know the exact input and output of a method, i.e., the signature. In my understanding of specs I would include signatures, because they help establish the contract of the method in a concise way. That might not be a practice here, so treat this as a just a suggestion.